src/app/public/modules/repeater/repeater.service.ts

```diff
@@ -3,15 +3,60 @@ import {
   Injectable
 } from '@angular/core';
 
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
 import {
   SkyRepeaterItemComponent
 } from './repeater-item.component';
 
 @Injectable()
 export class SkyRepeaterService {
+
+  public activeItemId: BehaviorSubject<string> = new BehaviorSubject(undefined);
+
   public itemCollapseStateChange = new EventEmitter<SkyRepeaterItemComponent>();
 
-  public onItemCollapseStateChange(item: SkyRepeaterItemComponent) {
+  public items: Array<SkyRepeaterItemComponent> = new Array<SkyRepeaterItemComponent>();
```

Comment:
Our other components tend to use the square brackets for the array signature, `SkyRepeaterItemComponent[]`.

Reply:
Done.